docs-v2/components/ArcadeEmbed.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const ArcadeEmbed = ({
```

Comment:
Since we use TypeScript components now, you can type the `{ src, title }` input as an `interface`. See e.g. the `Callout.tsx` file in the same dir.

```tsx
interface CalloutProps {
  type?: "default" | "info" | "warning" | "error";
  emoji?: string | React.ReactNode;
  children: React.ReactNode;
}

const PipedreamCallout: React.FC<CalloutProps> = ({
  type,
  emoji,
  children,
}) => {
```